plugins/main/public/components/common/welcome/components/mitre_top/mitre-top.tsx

```diff
@@ -108,25 +114,48 @@ const MitreTopTacticsTechniques = ({
       setShowTechniqueDetails('');
     };
 
-    const openDiscover = (e, techniqueID) => {
-      NavigationService.getInstance().navigateToModule(e, 'overview', {
-        tab: 'mitre',
-        tabView: 'discover',
-        filters: { 'rule.mitre.id': techniqueID },
+    const goToDashboardWithFilter = async (e, techniqueID) => {
+      const indexPatternId = AppState.getCurrentPattern();
```

Comment:
**suggestion**: this should be using the data source instead of trusting the index pattern ID is obtained through the `AppState` service.

Reply:
The data source instance is created in the events table that is rendered by the accordion that contains the links. This could imply a refactor to extract the instance of the data source or communicate to the parents. So we could use this approach for now.